cirq/sim/mux.py

```diff
@@ -183,3 +200,74 @@ def sample_sweep(program: Union[circuits.Circuit, schedules.Schedule],
                               seed=prng)
         trial_results.append(measurements)
     return trial_results
+
+
+def final_density_matrix(
+        program: Union[circuits.Circuit, ops.Gate, ops.OP_TREE, schedules.
```

Comment:
`program: 'cirq.CIRCUIT_LIKE'`

If we don't already have that exposed as a constant, define it equal to the given expression and expose it.

Reply:
Done.